docs/development.rst

```diff
@@ -8,6 +8,29 @@ Quickstart to run MapEntity in development :
     $ docker-compose run --rm web ./manage.py createsuperuser
     $ docker-compose up
 
+Develop with docker::
+
+    $ docker-compose up
+
+Get Screamshoter and convertit working:
+
+To get screamshotter container working, it needs to access to your web container by using your browser url.
+
+If you use http://localhost:8000 to develop, it tries to access to itself to make web capture of application.
```

Comment:
If you use http://localhost:8000/ for development, it attempts to access itself to capture the web application.